app/views/projects/show.html.erb

```diff
@@ -66,10 +61,15 @@
                           </div>
                         </div>
                       <% end %>
-                      <%= link_to project_story_path(@project.id, story, format: "json"), method: "delete",  data: { confirm: 'Are you sure?', story_id: story.id }, class: "delete magenta", remote: true, title: "Delete" do %>
-                        <i class="fa fa-trash-o"></i>
-                        <span>Delete</span>
+
+                      <%= link_unless_archived(@project, "Delete", project_story_path(@project.id, story, format: "json"), color: "delete", icon: 'trash-o', method: :delete, remote: true, data_attr: { confirm: 'Are you sure?', story_id: story.id }) %>
+
+                      <% link_to new_project_story_path(@project.id, story_id: story.id), class: "clone", title: "Clone" do %>
```

Comment:
This link_to should be removed, it's a duplicate of the next link_unless_archived